README.md

```diff
@@ -107,6 +108,30 @@ const logger = loggerFactory.init({
 
 ```
 
+### multi streams
+
+To be compatible with previous versions. When there is `logStream` key in config, multi streams config key `logStreams` would be dismissed.
```

Comment:
not really need to keep the backward compatible, we can bump up the version according to https://semver.org/, it can reduce a lot of code complexity.